ENIGMAsystem/SHELL/Platforms/SDL/Gamepad.cpp

```diff
@@ -30,7 +30,9 @@ void cleanupGamepads() {
 }
 
 void pushGamepads() {
-  for (int i = 0; i < SDL_NumJoysticks(); ++i) gamepads[i].push();
+  for (int i = 0; i < SDL_NumJoysticks(); ++i) {
+    if (SDL_IsGameController(i)) gamepads[i].push();
+  }
```

Comment:
Might want to ask @time-killer-games about this one.

Reply:
this was causing a crash on android because I dont push non controllers to the gamepads vector on init